client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -112,7 +115,7 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
 
           <Box textAlign="center">
             <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
-              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Create'}
+              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'SIGN UP'}
```

Comment:
Please avoid using CAPITALCASE text as some screen readers interpret them as abbreviations, you can use a CSS class to transform text instead.